Makefile

```diff
@@ -249,6 +249,13 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man
 
 varlink_generate: .gopathok cmd/podman/varlink/ioprojectatomicpodman.go
 
+.PHONY: install.libseccomp.sudo
+install.libseccomp.sudo:
+	rm -rf ../../seccomp/libseccomp
+	git clone https://github.com/seccomp/libseccomp ../../seccomp/libseccomp
+	cd ../../seccomp/libseccomp && git checkout $(LIBSECCOMP_COMMIT) && ./autogen.sh && ./configure --prefix=/usr && make all && sudo make install
```

Comment:
Need to define $LIBSECCOMP_COMMIT somewhere - I suggest whatever's the most recent stable tag?